cdap-docs/build.sh

```diff
@@ -460,18 +490,105 @@ function copy_docs_inner_level() {
 }
 
 function build_zip() {
+  local zip_target=${1}
+  local zip_extras=${2}
   set_project_path
-  make_zip ${1}
-}
-
-function zip_extras() {
-  if [[ "x${1}" == "x" ]]; then
-    return
+  set_version
+  if [ "x${1}" == "x" ]; then
+    ZIP_DIR_NAME="${PROJECT}-docs-${PROJECT_VERSION}"
+  else
+    ZIP_DIR_NAME="${PROJECT}-docs-${PROJECT_VERSION}-$1"
```

Comment:
Use `${zip_target}` here. Also, don't use string comparison to determine if a variable is set...
`if [[ -n ${zip_target} ]]; then`